bin/main.ml

```diff
@@ -1073,14 +1063,23 @@ let get_libdir context ~libdir_from_command_line =
   | Some p -> Fiber.return (Some (Path.of_string p))
   | None -> Context.install_ocaml_libdir context
 
+let print_unix_error f =
+  try
+    f ()
+  with Unix.Unix_error (e, _, _) ->
+    Format.fprintf err_ppf "@{<error>Error@}: %s@."
```

Comment:
Out of curiosity, why is it not possible to print on stderr directly?

Reply:
Isn't this better to do it this way so that buffering is consistent?

Reply:
This is a synchronous call so it's not an issue